Analysis

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+class Analysis:
+    @staticmethod
+    def brightness_trace(slice):
```

Comment:
Could you also add a multiline comment to this method as well? Basically the same thing as the comment I left on the `Analysis` class